src/DistributedLock.Azure/packages.lock.json

```diff
@@ -47,14 +47,6 @@
           "System.IO.Hashing": "6.0.0"
         }
       },
-      "Microsoft.Bcl.AsyncInterfaces": {
-        "type": "Transitive",
```

Comment:
Manually confirmed that there is a corresponding NuGet package with `type` `CentralTransitive` here and elsewhere.